cmd/importcorpora/main.go

```diff
@@ -0,0 +1,203 @@
+// Usage: go run cmd/importcorpora/main.go
+//
+// Updates the at the bottom of this file specified data files with content from dariusk/corpora.
+package main
```

Comment:
I think we can also at a task to the Makefile. Then it will be easier to use it.